Source/WebKit/Platform/cocoa/CocoaHelpers.mm

```diff
@@ -357,6 +361,62 @@ id parseJSON(API::Data& json, JSONOptionSet options, NSError **error)
     return [NSError errorWithDomain:error.domain ?: @"" code:error.code userInfo:nil].description;
 }
 
+NSURL *ensureDirectoryExists(NSURL *directory)
+{
+    ASSERT(directory.isFileURL);
+    NSError *error;
+    if (![NSFileManager.defaultManager createDirectoryAtPath:directory.path withIntermediateDirectories:YES attributes:nil error:&error])
+        return nil;
+    return directory;
+}
```

Comment:
We should use `FileSystem::makeAllDirectories` from `#import <wtf/FileSystem.h>` for this.